Source/Urho3D/CMakeLists.txt

```diff
@@ -162,7 +162,22 @@ set (THIRD_PARTY_DEPENDENCIES
     Recast
     Box2D
     WebP
-    Bullet
+    
+    dAnimation
+    dContainers
+    dCustomJoints
+    dgCore
+    dgNewtonAvx
+    dgNewtonAvx2
+    dgNewtonSse4.2
+    dgPhysics
+    dNewton
+    dMath
+    dVehicle
+    newton
+    dScene
+    tinyxml
```

Comment:
Actually SDK does not depend on tinyxml, just samples, so its not needed here.

Reply:
I thought I remember newtons scene class or something needing tinyxml, that was the only reason I thought I had to use it.  removing it would be great